tree/dataframe/inc/ROOT/RDFNodes.hxx

```diff
@@ -82,7 +82,8 @@ using RangeBaseVec_t = std::vector<RangeBasePtr_t>;
 class RLoopManager {
    using RDataSource = ROOT::RDF::RDataSource;
    enum class ELoopType { kROOTFiles, kROOTFilesMT, kNoFiles, kNoFilesMT, kDataSource, kDataSourceMT };
-
+   // Needed to allow the RJittedFilter to trigger JITting
+   friend class RJittedFilter;
```

Comment:
let's make `BuildJittedNodes` public in `RLoopManager` instead: it seems a lot of entities need to call it